resources/puppet-webhook.service

```diff
@@ -1,13 +1,11 @@
 [Unit]
+Description=Puppet Webhook API Server
 After=network.target
 
 [Service]
-EnvironmentFile=-/etc/sysconfig/puppet-webhook
-EnvironmentFile=-/etc/default/puppet-webhook
-WorkingDirectory=/opt/voxpupuli/webhook
-ExecStart=/opt/voxpupuli/webhook/bin/bundle exec puma config.ru
-ExecReload=/bin/kill -HUP $PID
-KillMode=process
+Type=oneshot
+ExecStart=/bin/true
```

Comment:
i am confused about what exactly is being one-shot started here

Reply:
no process. but this unit triggers puppet-webhook-sidekiq and puppet-webhook-app

Reply:
ah, by those others being PartOf, now i see